src/auditwheel/main_repair.py

```diff
@@ -85,6 +85,12 @@ def configure_parser(sub_parsers):
         help="Strip symbols in the resulting wheel",
         default=False,
     )
+    p.add_argument(
+        "--exlude",
+        dest="EXCLUDE",
+        help="List of excluded so files as a comma-separated string",
```

Comment:
Not sure about the plural of `soname` or using this directly as a noun but it disambiguates wether we're talking file names/file paths or soname.
```suggestion
        help="List of excluded sonames as a comma-separated string",
```